src/Components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => (
+  <nav className="navbar">
+    <div className="container">
+          <NavLink
```

Comment:
```suggestion
    <div className="container">
      <NavLink
```
Pay attention to indentations